terraform-google-{{cookiecutter.module_name}}/test/make.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
```

Comment:
I think that we should note that this file another other scaffolded helpers are generated by a cookiecutter template, so that code reviewers know to not pick over this file during the review process. Thoughts?